noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -1367,10 +1749,14 @@
                 ]]>
             </warnings>
         </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST15">
         <title>Document Description period containment</title>
+        <description>
+            Tests whether the Document Descriptions' created and finalized dates
```

Comment:
`Tests whether the created and finalized dates of all document descriptions are within the archival period specified in arkivuttrekk.xml`